package.json

```diff
@@ -6,6 +6,7 @@
   "main": "./lib",
   "scripts": {
     "build": "babel ./src --out-dir ./lib",
+    "build:icon-index-generator": "cd src/svg-icons/ && babel-node index-generator.js",
```

Comment:
Can we change this to just `build:icons`? Not sure we'll ever need that `npm scripts` namespace for anything else